generators/kubernetes/templates/deployment.yml.ejs

```diff
@@ -72,14 +71,14 @@ spec:
           value: prod
         <%_ if (app.serviceDiscoveryType === 'eureka') { _%>
         - name: SPRING_CLOUD_CONFIG_URI
-          value: http://admin:${jhipster.registry.password}@jhipster-registry.<%= kubernetesNamespace %>.svc.cluster.local:8761/config
+          value: http://admin:${jhipster.registry.password}@jhipster-registry:8761/config
```

Comment:
would this still work if I specify a namespace during deployment?

Reply:
of course